poedit.appdata.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2015 Mario Blättermann <mariobl@fedoraproject.org> -->
+<application>
+  <id type="desktop">poedit.desktop</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>MIT</project_license>
+  <name>Poedit</name>
+  <name xml:lang="de">Poedit</name>
```

Comment:
Is this necessary, e.g. for the description to be picked up? Seeing as the name is going to be “Poedit” in all languages, it seems pointless...